README.md

```diff
@@ -1,13 +1,14 @@
 # knapsack
-Pseudo-polynomial approach to Knapsack Problem
 
-I thought I might write one algorithm that I find really interesting: Knapsack problem.
-
-Its problem statement is simple:
+Knapsack is a Pseudo-polynomial approach to Knapsack Problem. Knapsack problem is a problem, which I personally interest me. I decided to write an algorithm to solve the problem.
```

Comment:
- _"Knapsack is a Pseudo-polynomial" : **Out of context**.  This will be right if you specify knapsack the repo, not the problem itself
- _"Knapsack problem is a problem"_ : **redundancy.**
- _"which I personally interest me"_ : **grammar error**